README.md

```diff
@@ -1,4 +1,14 @@
 # GitHub Activity Checker
-> Check your public activity on GitHub!
 
-This is a simple Website that shows your an user's recent public activity on GitHub using GitHub API. The UI is implemented using materializecss.
+This website uses the GitHub API to scrape your GitHub profile to show your GitHub activity.
+
+# Features
+
+- Instant search
+- Shows profile info like bio, company, location, etc.
+- Shows all your repos.
+- Shows in a calender view the activity on GitHub.
+
+# Contributing
```

Comment:
There is already a CONTRIBUTING.md file. No need of this section.

Reply:
Fixed